openedx/core/djangoapps/bookmarks/models.py

```diff
@@ -60,6 +61,28 @@ def __unicode__(self):
         return self.resource_id
 
     @classmethod
+    def can_create_more(cls, data):
```

Comment:
This function should be in `api.py`. It does not have anything to do with the model.